docker-compose.dev.yml

```diff
@@ -3,8 +3,6 @@ services:
   api:
     environment:
       - API_AUTH_TOKEN_SECRET=dont_tell_anyone
-    ports:
-      - "3000:3000"
```

Comment:
I sometimes used `./bin/start` after `./bin/dev` to quickly manually test. Now that the pipeline is set up we probably do not need it anymore.